### DISCUSSION TITLE
Please add sonar transducer radius

### DISCUSSION THREAD

**Discussion by HardRedux** [FakeFishGames/Barotrauma#14046] [Ideas and suggestions]
Please add sonar transducer radius
I am writing through a translator. Dear developers, please make it possible to change the radius of the sonar transducer signal in the submarine editor. It cannot be changed now. For example, I want to make a torpedo but I can’t, 100 meters is a very small radius. After **100** meters it says "no signal"
![Screenshot_62](https://github.com/FakeFishGames/Barotrauma/assets/138946919/53d192f7-2880-4ab2-b0f1-80f28454903c)

**Comment by SomeRandomNoobKekeke** [comment]
You can set signal radius for wifi components

**Reply by HardRedux** [reply]
The radius of the Wi-Fi components costs 90000 :(

**Reply by SomeRandomNoobKekeke** [reply]
You can set any radius for wifi component, idk what you are talking about
You need to set radius for both wifi components or the one with smaller radius will limit you, obviously
![20240529134812_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/9230ab94-d60b-47c0-87ad-4a37d96ce4ef)
![20240529135158_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/b9dd8918-4d2d-495e-acd6-487d37fbb634)

**Reply by HardRedux** [reply]
Thanks a lot! My mistake 😅 Indeed, one Wi-Fi component responsible for the sonar transducer was set to 20000. What a kind community, they even showed it in the photo 😇
